flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/descriptors/RegistrableDescriptor.scala

```diff
@@ -1,49 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.flink.table.descriptors
-
-/**
-  * A trait for descriptors that allow to register table source and/or sinks.
-  */
-trait RegistrableDescriptor extends Descriptor {
```

Comment:
Why couldn't we move those classes (also e.g. TableFactoryUtil) in the previous commit?

Reply:
I will move this change into the right commit.